src/cc_catalog_airflow/dags/util/loader/sql.py

```diff
@@ -246,3 +265,25 @@ def _get_load_table_name(
         load_table_name_stub=LOAD_TABLE_NAME_STUB,
 ):
     return f'{load_table_name_stub}{identifier}'
+
+
+def _get_malformed_row_in_file(error_msg):
+    error_list = error_msg.splitlines()
+    copy_error = error_list[3]
+    assert copy_error.startswith('COPY'), copy_error
+
+    copy_error_info_list = copy_error.split(',')
+    line_number = int(copy_error_info_list[1].split(' ')[-1])
```

Comment:
I'd prefer the following:
```suggestion
    line_number = int(copy_error.split('line ')[1].split(',')[0])
```
This grabs whatever is after the first space after the first instance of the string `'line'`, and before the next comma.  I have more confidence in the pattern `line X,` staying consistent than counting commas.